docs-src/ref-programs-step.scrbl

```diff
@@ -247,7 +247,7 @@ fork()
 }
```

Comment:
The revert is going to be a pain. I would do an interactive rebase and delete the commit